core/math/geometry_2d.cpp

```diff
@@ -351,3 +376,494 @@ Vector<Vector3i> Geometry2D::partial_pack_rects(const Vector<Vector2i> &p_sizes,
 
 	return ret;
 }
+
+static void tessellate_cubic_bezier_in_rect(Vector<Vector2> &r_out, Vector2 p_start, Vector2 p_control1, Vector2 p_control2, Vector2 p_end, Vector2 p_start_transformed, Vector2 p_control1_transformed, Vector2 p_control2_transformed, Vector2 p_end_transformed, const Rect2 &p_limit) {
+	while (true) {
+		// Stop condition - Completely out of bounds
```

Comment:
```suggestion
		// Stop condition - Completely out of bounds.
```
As per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide), same elsewhere (including code you've just copied)